### PR TITLE
*: cut v1.4.0

### PR DESCRIPTION
**Pull request by brancz**
After a testing period of 16 days, there were no additional bugs found or features introduced.

@andyxning @zouyee 

cc @mxinden 

@loburm @piosz please stand by to push the final v1.4.0 image to gcr.io once this PR is merged and the tag is pushed.

### CODE REVIEW

File: kubernetes/kube-state-metrics-deployment.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: kube-state-metrics
       containers:
       - name: kube-state-metrics
-        image: quay.io/coreos/kube-state-metrics:v1.3.1
+        image: quay.io/coreos/kube-state-metrics:v1.4.0
```
Comment (zouyee) [inline kubernetes/kube-state-metrics-deployment.yaml]: bump addon-resizer image version at the same time?

Comment (andyxning) [reply kubernetes/kube-state-metrics-deployment.yaml]: Another PR should be introduced to do this.

**Comment by brancz** [comment]
@loburm @piosz the release is tagged and the container image is pushed to quay.

**Comment by loburm** [comment]
@brancz done, sorry for the delay. Image should be available in gcr soon.

**Pull request merged**